front/lib/api/assistant/conversation/mentions.test.ts

```diff
@@ -1487,18 +1480,12 @@ describe("createUserMessage", () => {
 
     // Create origin message
     const originMessage = await withTransaction(async (transaction) => {
-      const originUserMessage = await UserMessage.create(
+      const originAgentMessage = await AgentMessage.create(
         {
-          content: "Origin message",
-          userId: user.id,
           workspaceId: workspace.id,
-          userContextUsername: userJson.username,
-          userContextTimezone: "UTC",
-          userContextFullName: userJson.fullName,
-          userContextEmail: userJson.email,
-          userContextProfilePictureUrl: userJson.image,
-          userContextOrigin: "web",
-          clientSideMCPServerIds: [],
+          skipToolsValidation: false,
+          agentConfigurationId: "not-a-real-agent",
```

Comment:
I would at least put that in a const, or as a special case in the type of `agentConfigurationId`